@stellar/anchor-tests/src/helpers/config.ts

```diff
@@ -164,6 +164,13 @@ function checkSepConfigObj(config: Config) {
       );
     }
   }
+  if (config.seps.includes(24)) {
+    if (!config.sepConfig || !config.sepConfig["24"]) {
+      throw new ConfigError(
+        "SEP 24 configuration is required to run SEP-24 tests.",
+      );
+    }
+  }
```

Comment:
Why do we need this check if the config file is optional in the UI?

Reply:
We can probably remove it, I'll take look